upgrade/sql/8.1.0.sql

```diff
@@ -29,8 +29,18 @@ CREATE TABLE `PREFIX_product_attribute_lang` (
 
 /* PHP:ps_810_add_product_attribute_lang_data(); */;
 
-/* Add default redirect configuration and change all '404' to 'default' */
-INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES  
-  ('PS_PRODUCT_REDIRECTION_DEFAULT', '404', NOW(), NOW());
+/* Add default redirect configuration and change all '404' to 'default'
+   as well as new image feature configuration
+ */
+INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES
+  ('PS_PRODUCT_REDIRECTION_DEFAULT', '404', NOW(), NOW()),
+  ('PS_AVIF_QUALITY', '90', NOW(), NOW()),
+  ('PS_IMAGE_FORMAT', 'jpg', NOW(), NOW());
+
 UPDATE `PREFIX_product` SET `redirect_type` = 'default' WHERE `redirect_type` = '404';
 UPDATE `PREFIX_product_shop` SET `redirect_type` = 'default' WHERE `redirect_type` = '404';
+
+/* add new feature flag for multiple image formats */
+INSERT INTO `PREFIX_feature_flag` (`name`, `state`, `label_wording`, `label_domain`, `description_wording`, `description_domain`)
```

Comment:
I believe the stability is missing unless it's still considered beta? Then it's fine because it's the default value anyway.